src/components/BeerListItem/BeerListItem.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ButtonBeerName from '../ButtonBeerName/ButtonBeerName';
+import beerSign from '../../images/empty-beer-photo.svg';
+
+function BeerListItem({ beer }) {
+  const beerImage = beer.image || beerSign;
+  const inStock = beer.amount > 0;
+  return (
+    <div className="BeerListBox">
```

Comment:
Please make full beerItem clickable to make it easier to click on mobile devices (and desktops)